File: python/p020.py
#!/usr/bin/python

total = 0
fact = 1
for x in range(1, 101):
    fact *= x

while fact > 0:
    x = fact % 10
    total += x
    fact = (fact - x) // 10

# total = sum([int(i) for i in str(fact)])

print(int(total))

File: rust/src/bin/p020.rs

fn main() {

    let mut total = 0;
    let mut fact = (1..11).fold(1, |acc, x| acc * x);

    while fact > 0 {
        let x = fact % 10;
        total += x;
        fact = (fact - x) / 10;
    }

    println!("{}", total);

}
